src/test/java/com/newestuser/imen/bgdate/LeapYearTest.java

```diff
@@ -0,0 +1,26 @@
+package com.newestuser.imen.bgdate;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class LeapYearTest {
+
+    @Test
+    public void findIsLeapYear() {
+        assertFalse(findIsLeapYear(1800));

```

Comment:
The idea of the test is to guarantee that when the implementation changes the behavior will remain the same.

Currently this test does not have a way to guarantee that. If I go and change the logic in BulgarianDate nothing will fail. 

The tes should test the logic in BulgarianDate not duplicate it. 

Also you can move this test method in the existing test class. For now there is no need to have a separate test class just for leap years.